drivers/cisco/booking_panel_led_sync.cr

```diff
@@ -0,0 +1,49 @@
+require "placeos-driver"
+
+class Cisco::BookingPanelLedSync < PlaceOS::Driver
+  descriptive_name "Cisco Webex Navigator Panel LED Sync"
+  generic_name :Navigator_LED_Sync
+  description "Sync Cisco Webex Navigator Panel LED to Bookings.in_use status"
+
+  default_settings({
+    led_color_when_room_booked: "Red",
+    led_color_when_room_available: "Green",
+    webex_panel_device_id: "Ensure this is set in each System's settings"
+  })
+
+  accessor webex_xapi : CloudXAPI
+  accessor room_bookings : Bookings
+
+  @led_color_when_room_booked : String = "Red"
+  @led_color_when_room_available : String = "Green"
+  @webex_panel_device_id : String = "Ensure this is set in each System's settings"
+
+  def on_load
+    on_update
+    sync_led_color_now
+  end
+
+  def on_update
+    clear_subscriptions
+    @led_color_when_room_booked = setting(:led_color_when_room_booked) || "Red"
+    @led_color_when_room_available = setting(:led_color_when_room_available) || "Green"
+    @webex_panel_device_id = setting(:webex_panel_device_id) || "Ensure this is set in each System's settings"
+
+    system.subscribe("Bookings_1", "in_use") do |_sub, value|
+      next unless ["true", "false"].includes?(value)
+      self[:room_in_use] = room_in_use = value == "true"
+      set_led_color(room_in_use)
+    end
+  end
+
+  def sync_led_color_now
+    return unless ["true", "false"].includes?(room_in_use = system[:Bookings][:in_use].as_bool)
```

Comment:
Driver Harness states that system[:Bookings][:in_use] is JSON::Any and errors with and withohut `.as_bool`
What's the best way to handle this to fetch the Bool value of another module's status?

Reply:
What's the actual error you are getting?

You can try inserting these lines above it to get more details about what `system[:Bookings][:in_use]` actually contains.
```
pp! system[:Bookings][:in_use]
pp! system[:Bookings][:in_use] == nil
pp! typeof(system[:Bookings][:in_use])
```